AndelaOSP/wire
# #152631364 Create Login Interface

#### What does this PR do?
Create Login Interface

#### Description of Task to be completed?
Create Login Interface

#### How should this be manually tested?
- Start the project 
- Access `http://localhost:8080/`
- You should see the login page.

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[152631364](https://www.pivotaltracker.com/story/show/152631364)

#### Screenshots (if appropriate)
<img width="1280" alt="screen shot 2017-11-13 at 17 18 36" src="https://user-images.githubusercontent.com/25595450/32730629-c985bd76-c898-11e7-9bae-a9ef8361c75f.png">

#### Questions:
N/A
